### DISCUSSION TITLE
how to use a dynamic value in field name with antlers?

### DISCUSSION THREAD

**Discussion by Ravenna** [statamic/cms#5262] [Q&A] [answered]
Is it possible to use a partial parameter in a field name in antlers?  
My use case: 
{{ partial src="some/partial" side="left" }} and then in the partial {{ {side}_kicker }}. To be clear {{ {side}_kicker }} doesn't work

**Comment by jackmcdade** [comment, marked as answer]
Alas, it is not possible. I'm sorry!

**Comment by Ravenna** [comment]
@jackmcdade if someone was to think about how to make that work: where would the start?

**Reply by jackmcdade** [reply]
I would try to restructure the data so you didn't have to assemble a variable variable name, honestly. Can you share what you're trying to do so I can make other suggestions?

**Reply by Ravenna** [reply]
@jackmcdade I came up with a different approach that was simpler.  Thanks

**Reply by jackmcdade** [reply]
Simpler is great!
